go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/kopia/kopia
 
-go 1.22.7
+go 1.22
```

Comment:
https://github.com/kaovilai/kopia/actions/runs/12367614994/job/34516203180?pr=2
Vuln check is passing.